src/seedu/addressbook/parser/Parser.java

```diff
@@ -120,6 +131,9 @@ public Command parseCommand(String userInput) {
         case CreateExamCommand.COMMAND_WORD:
             return prepareCreateExam(arguments);
 
+        case AddAssignmentStatistics.COMMAND_WORD:
```

Comment:
Inconsistent use of verbs

Reply:
Inconsistent in what way?

Reply:
The command is AddAssignmentStatistics, but you prepareCreateStatistics